lib/main-menu.js

```diff
@@ -136,6 +143,33 @@ var view = {
       click: function () {
         BrowserWindow.getFocusedWindow().toggleDevTools()
       }
+    },
+    {
+      type: 'separator'
+    },
+    {
+      label: 'Next Note',
+      accelerator: 'Control + J',
+      click: function () {
+        mainWindow.webContents.send('list:next')
+      }
+    },
+    {
+      label: 'Prior Note',
```

Comment:
I think `prev` is more famous. (But I'm not good at writing English. So if I was wrong, tell me please :bow: )